arroyo-controller/src/lib.rs

```diff
@@ -414,6 +416,28 @@ impl ControllerGrpc for ControllerServer {
 
         Ok(Response::new(ReceiverStream::new(rx)))
     }
+
+    async fn worker_error(
+        &self,
+        request: Request<WorkerErrorReq>,
+    ) -> Result<Response<WorkerErrorRes>, Status> {
+        info!("Got worker error.");
+        let req = request.into_inner();
+        let client = self.db.get().await.unwrap();
+        let _ = queries::controller_queries::create_job_log_message()
+            .bind(
+                &client,
+                &req.job_id,
+                &LogLevel::error,
+                &req.message,
+                &req.details,
+            )
+            .one()
+            .await
+            .unwrap();
```

Comment:
This should probably return an error back to the client rather than panicking

Reply:
But if we're preserving the original behavior of the worker panicking, then what's the point of handling the error of failing to write the error message?

Reply:
This is within the controller, not the worker. The controller is responsible for all jobs, so shouldn't panic.

Reply:
Gotcha. Ok I made it return an error using `Status::from_error`. Is that good or should we force it to be a specific error like `Status::Internal`?

Reply:
I would create a Status::Internal error with an appropriate message. It doesn't matter that much here, since the consumers are internal, but it's a good practice.